src/ralph/backends/storage/swift.py

```diff
@@ -32,16 +27,20 @@ class SwiftStorage(
 
     def __init__(
         self,
-        os_tenant_id,
-        os_tenant_name,
-        os_username,
-        os_password,
-        os_region_name,
-        os_storage_url,
-        os_user_domain_name=SWIFT_OS_USER_DOMAIN_NAME,
-        os_project_domain_name=SWIFT_OS_PROJECT_DOMAIN_NAME,
-        os_auth_url=SWIFT_OS_AUTH_URL,
-        os_identity_api_version=SWIFT_OS_IDENTITY_API_VERSION,
+        os_tenant_id: str = settings.BACKENDS.STORAGE.SWIFT.OS_TENANT_ID,
+        os_tenant_name: str = settings.BACKENDS.STORAGE.SWIFT.OS_TENANT_NAME,
+        os_username: str = settings.BACKENDS.STORAGE.SWIFT.OS_USERNAME,
+        os_password: str = settings.BACKENDS.STORAGE.SWIFT.OS_PASSWORD,
+        os_region_name: str = settings.BACKENDS.STORAGE.SWIFT.OS_REGION_NAME,
+        os_storage_url: str = settings.BACKENDS.STORAGE.SWIFT.OS_STORAGE_URL,
+        os_user_domain_name: str = settings.BACKENDS.STORAGE.SWIFT.OS_USER_DOMAIN_NAME,
+        os_project_domain_name: str = (
+            settings.BACKENDS.STORAGE.SWIFT.OS_PROJECT_DOMAIN_NAME
```

Comment:
```suggestion
            swift_settings.OS_PROJECT_DOMAIN_NAME
```
It's quite verbose here, could we use an alias? or do we keep the same settings pattern everywhere?

Reply:
By an alias you mean something like the following?

```python
# swift.py
from ralph.conf import settings

swift_settings = settings.BACKENDS.STORAGE.SWIFT
```

If yes, I'm totally fine with it.

Reply:
Thanks) updated this)